seatunnel-connectors-v2/connector-neo4j/src/main/java/org/apache/seatunnel/connectors/seatunnel/neo4j/config/Neo4jSinkConfig.java

```diff
@@ -29,4 +29,17 @@ public class Neo4jSinkConfig extends Neo4jCommonConfig {
                     .noDefaultValue()
                     .withDescription(
                             "position mapping information for query parameters. key name is parameter placeholder name. associated value is position of field in input data row.");
+
+    public static final Option<Integer> MAX_BATCH_SIZE =
+            Options.key("maxBatchSize")
+                    .intType()
+                    .noDefaultValue()
+                    .withDescription("neo4j write max batchSize");
+
+    public static final Option<String> BATCH_VARIABLE =
+            Options.key("batchVariable")
+                    .stringType()
```

Comment:
```suggestion
                    .enumType()
```

Reply:
Remove?

Reply:
batch_data_variable represents how to represent a batch of data in a cypher statement, usually using $batch, but here I still leave it open to the user to select the configuration